### PR TITLE
feat: add mixpanel

### PR DESCRIPTION
**Pull request by bigint**
CodeSandbox logoCodeSandbox logo&nbsp; Open in CodeSandbox <a href="https://codesandbox.io/p/github/lensterxyz/lenster/mixpanel?workspace=%7B%22gitSidebarPanel%22:%22PR%22,%22sidebarPanel%22:%22GIT%22%7D">Web Editor</a> | <a href="https://codesandbox.io/p/vscode?owner=lensterxyz&repo=lenster&branch=mixpanel">VS Code</a>

### CODE REVIEW

File: src/components/SiteLayout.tsx [hunk 1]
```diff
@@ -85,6 +91,13 @@ const SiteLayout: FC<Props> = ({ children }) => {
     const currentUserAddress = currentUser?.ownedBy
     setMounted(true)
 
+    // Set mixpanel user id
+    if (currentUser?.id) {
+      Mixpanel.identify(currentUser?.id)
+    } else {
+      Mixpanel.identify('0x00')
+    }
```
Comment (harish-sethuraman) [inline src/components/SiteLayout.tsx:99]:
```suggestion
   Mixpanel.identify(currentUser?.id || '0x00')
```
may be? BTW don't chain same path twice

File: src/lib/mixpanel.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import mixpanel from 'mixpanel-browser'
+
+const enabled = true
+
+const actions = {
+  identify: (id?: string) => {
```
Comment (harish-sethuraman) [inline src/lib/mixpanel.ts:6]: Why is id optional? I see we are setting values however?

Comment (bigint) [reply src/lib/mixpanel.ts:6]: Copilot 🤣

File: src/components/SiteLayout.tsx (hunk 1 above)
Comment (bigint) [reply src/components/SiteLayout.tsx:99]: We will set more user data if logged in!

File: src/lib/mixpanel.ts (hunk 2 above)
Comment (bigint) [reply src/lib/mixpanel.ts]: Now i wrote good comment and copilot worked correctly 😅 

pushed new changes

Comment (harish-sethuraman) [reply src/lib/mixpanel.ts]: Okay boss

**Pull request merged**